controller/controller.js

```diff
@@ -9,8 +9,22 @@ fetchCategories().then((result) => {
 
 function getReviews(request, response, next) {
     fetchReviews().then((result) => {
-        response.status(200).send(result)
+        response.status(200).send({reviews: result})
     }).catch(next)
 }
 
-module.exports = {getCategories, getReviews}
\ No newline at end of file
+function getReviewsByReviewId(request, response, next) {
+    const {review_id} = request.params
+    Promise.all([fetchReviewsByReviewId(review_id)])
```

Comment:
Shouldn't need to use Promise.all here, you're passing it an array with a single value in it, which would be equivalent to just having
`fetchReviewsByReviewId(review_id).then(...` etc.